config/plugins.yaml

```diff
@@ -133,6 +138,8 @@ milestone_applier:
     master: v0.0
   cert-manager/policy-approver:
     master: v0.0
+  cert-manager/trust:
+    master: v0.0
```

Comment:
hmm.. is `v0.0` correct here? I see it for istio-csr & policy-approver too, but that's surely never a milestone we'd use?

Reply:
No, that is intended as a placeholder but istio-csr at least hasn't been updated :sweat_smile: 

Should this be removed completely?